py3status/modules/mpris.py

```diff
@@ -123,17 +125,19 @@ class Py3status:
     button_next = 4
     button_previous = 5
     buttons_order = 'previous,toggle,next'
+    color_buttons_bad = None
+    color_buttons_good = None
```

Comment:
maybe `color_control_active` /  `color_control_inactive` buttons is a little confusing imho